README.md

```diff
@@ -12,20 +12,29 @@ Stable binaries are also available on the [releases](https://github.com/viveksyn
 ## For Linux:
 ```
 curl -Lo ./kube-sync https://github.com/viveksyngh/kube-sync/releases/download/v0.1.0/kube-sync-$(uname)-amd64
+```
```

Comment:
What is the benefit on putting them in separate code block ? The reason they were put in same code block because then can be copied and pasted in one go.

Reply:
I agree that If we put all commands in one block then it can be copied and pasted in one go.

But I suggested this change from a user perspective, When I followed the documentation I realize that executing command will be more convenient If executed one by one. Hence I put each command in a separate block. 

Let me know If you still want me to revert this change.

Reply:
I would like to keep this same as it is for now. May be later we can revisit this after more feedback.